LocalSettings.php

```diff
@@ -1949,16 +1949,39 @@
 				'isLocal' => false,
 			],
 		],
+		'gpcommonswiki' => [
+			'Tabular.JsonConfig' => [
+				'namespace' => 486,
+				'nsName' => 'Data',
+				// page name must end in ".tab", and contain at least one symbol
+				'pattern' => '/.\.tab$/',
+				'license' => 'CC0-1.0',
+				'isLocal' => false,
+			],
+			'Map.JsonConfig' => [
+				'namespace' => 486,
+				'nsName' => 'Data',
+				// page name must end in ".map", and contain at least one symbol
+				'pattern' => '/.\.map$/',
+				'license' => 'CC0-1.0',
+				'isLocal' => false,
+			],
+		],
 	],
 	'wgJsonConfigInterwikiPrefix' => [
 		'default' => 'commons',
 		'commonswiki' => 'meta',
+		'gpcommonswiki' => 'c',
 	],
 	'wgJsonConfigModels' => [
 		'default' => [
 			'Map.JsonConfig' => JsonConfig\JCMapDataContent::class,
 			'Tabular.JsonConfig' => JsonConfig\JCTabularContent::class,
 		],
+		'gpcommonswiki' => [
+			'Tabular.JsonConfig' => 'JsonConfig\JCTabularContent',
+			'Map.JsonConfig' => 'JsonConfig\JCMapDataContent',
+		],
```

Comment:
This is the exact same as the above default.

Reply:
> This is the exact same as the above default.

Oh, I'd remove that then, I thought there was a difference between both, like if they function differently.

Reply:
Nope, its the exact same as the default.

Reply:
> Nope, its the exact same as the default.

Okay, I'd just remove it then.

Reply:
Please do then.